metrics/ttfp/run-benchmark.jl

```diff
@@ -13,28 +32,90 @@ function github_context()
     )
 end
 
-function make_or_edit_comment(ctx, pr, comment)
+function prettytime(trial)
+    t = time(median(trial))
+    # Taken from Benchmarktools, since we only want two digits
+    if t < 1e3
+        value, units = t, "ns"
+    elseif t < 1e6
+        value, units = t / 1e3, "μs"
+    elseif t < 1e9
+        value, units = t / 1e6, "ms"
+    else
+        value, units = t / 1e9, "s"
+    end
+    return string(@sprintf("%.2f", value), units)
+end
+
+function speedup(t1, t2)
+    t = judge(median(t2), median(t1))

```

Comment:
would be better to be specific about the tolerance used, otherwise it's not better than comparing two point estimates directly, but without regard for the variance